### PR TITLE
Add address enable/disable function

### PR DESCRIPTION
**Pull request by shekhar-cis**
I have added function to `enable and disable` identity in `src/addresses.py`.

### CODE REVIEW

File: src/addresses.py [hunk 1]
```diff
@@ -281,3 +282,15 @@ def addBMIfNotPresent(address):
     """Prepend BM- to an address if it doesn't already have it"""
     address = str(address).strip()
     return address if address[:3] == 'BM-' else 'BM-' + address
+
+
+def disable_addresses(address):
```
Comment (PeterSurda) [inline src/addresses.py:287]: should be signular as it only disables one address

File: src/addresses.py [hunk 2]
```diff
@@ -281,3 +282,15 @@ def addBMIfNotPresent(address):
...
+    """"To disable the Address"""
+    BMConfigParser().set(str(address), 'enabled', 'false')
+    BMConfigParser().save()
+
+
+def enable_addresses(address):
```
Comment (PeterSurda) [inline src/addresses.py:293]: here too

File: src/addresses.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
 from binascii import hexlify, unhexlify
 from struct import pack, unpack
 
+from bmconfigparser import BMConfigParser
```
Comment (PeterSurda) [inline src/addresses.py:10]: since this adds an import, it may be better to put the functions into `BMConfigParser` directly and make them methods

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/addresses.py (hunk 3 above)
Comment (shekhar-cis) [reply src/addresses.py:10]: Alright, i'll create a new PR to do update in `BMConfigparser`.

**Pull request closed without merging**